lighthouse-core/lib/lh-error.js

```diff
@@ -5,7 +5,40 @@
  */
 'use strict';
 
-const strings = require('./strings');
+const i18n = require('./i18n/i18n.js');
+
+/* eslint-disable max-len */
+const UIStrings = {
+  /** Error message explaining that the Lighthouse run was not able to collect screenshots through Chrome.*/
+  didntCollectScreenshots: `Chrome didn't collect any screenshots during the page load. Please make sure there is content visible on the page, and then try re-running Lighthouse.`,
+  /** Error message explaining that the network trace was not able to be recorded for the Lighthouse run. */
```

Comment:
maybe for some of these descriptions we should include for the translators the distinction of whether it was something weird that retrying should fix, or something that the user needs to fix and try again? I'm thinking a stock explanation.

`This is an error message that was because of certain user conditions they must fix before retrying.`
vs.
`This is an error message that was because of irregularities with Lighthouse that should be fixed by trying again.`

just spitballin'